pool/web/view/page/page_layout.ml

```diff
@@ -130,33 +137,51 @@ module Tenant = struct
          tenant_languages)
   ;;
 
-  (* TODO[timhub]: * differ between login status *)
-  let navigation layout_context language query_language active_navigation =
+  let navigation
+    user
+    language
+    query_language
+    active_navigation
+    tenant_languages
+    active_lang
+    =
+    let open Pool_common.I18n in
+    let open Pool_context in
+    let to_main_nav = to_main_nav language query_language active_navigation in
+    let language_switch () = i18n_links tenant_languages active_lang in
```

Comment:
Does this need the `()` arg?

Reply:
No, and I also realised that I added it to the wrong navigation.